src/olympia/addons/tests/test_views.py

```diff
@@ -1316,14 +1316,28 @@ def test_set_homepage_support_url_email(self):
         }
 
         data = {
-            'homepage': {'ro': settings.EXTERNAL_SITE_URL},
-            'support_url': {'fr': f'{settings.EXTERNAL_SITE_URL}/foo/'},
+            'homepage': {'en-US': f'https://{settings.DOMAIN}'},
+            'support_url': {'en-US': f'https://{settings.DOMAIN}/foo/'},
```

Comment:
same question here on changing to `en-US` (and so on in this file)

Reply:
As above